### DISCUSSION TITLE
Enhancement Request: Mirror audio from one source to two outputs

### DISCUSSION THREAD

**Discussion by DDubbleDD** [File-New-Project/EarTrumpet#661] [Ideas]
This issue specifically comes up when I am using my Samsung Windows Mixed Reality Headset.  Audio is normally only sent to the USB 2.0 Soundcard for the Headset.  We always mirror the Headset **display** over HDMI to a TV.  However, the **audio** is not mirrored through the HDMI to the TV.  There are some work-arounds that sometimes work using the current Windows Audio system, but it is Flaky.  I have gotten this to work with VB-Audio VoiceMeeter Banana but it is complicated.  When I use the WMR headset with the Steam app, Steam has an option to mirror audio to both the headset and to another  output which works wonderfully.  Can you add the capability to mirror a single source to two outputs in EarTrumpet?

**Comment by DDubbleDD** [comment]
This might be fixed in the Mixed Reality App in the next version of Windows Redstone 5.
From Thurrott.com:
"Windows Mixed Reality improvements. Windows Mixed Reality no longer requires a physical display, which is useful for backpack gaming PCs. And you can now capture images of the mixed reality world using the system capture experience, **plus stream audio to both your headset and the PC’s speakers simultaneously.**"

In my case I want to stream both to the USB audio of my Samsung HMD and the HDMI through the NVIDIA GTX1070.  I hope that is an option as well as "PC Speakers".

**Comment by riverar** [comment]
Interesting, will investigate this new functionality in Windows Mixed Reality.

**Comment by DDubbleDD** [comment]
I played with the new Mirror Audio setting in the WMR settings this morning in the new Windows 10 1809  and still could not get this to work to mirror audio between the Samsung Headset USB audio and the HDMI to the TV.  It doesn't even seem to mirror audio to the PC Speakers.  I'll play with it some more, but this is really disheartening that Microsoft can't seem to make this simple thing work seamlessly.  Steam did it right, even with the WMR headset...  So the request still stands for EarTrumpet to make up for Microsoft's failings in this regard.

**Comment by riverar** [comment]
Yeah, we've investigated this and it's tied to Mixed Reality headsets and the current audio renderer on the PC. There's no way around it.
